src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -1585,6 +1585,11 @@ private function addMailerSection(ArrayNodeDefinition $rootNode)
                                 ->end()
                             ->end()
                         ->end()
+                        ->arrayNode('notification')
+                            ->children()
+                                ->scalarNode('theme')
```

Comment:
This configuration may not be necessary if we promote the current Twig override mechanism, e.i:
```yaml
twig:
    paths:
        templates/email: email
```
this would be enough to customize also other templates such as `@email/zurb_2/notification.css` or `'@email/zurb_2/notification_body_markdown.html.twig'`, wdyt?

Reply:
Changing the default template used for notification emails has a big difference compared to overriding the templates: if you change the template being used, your own template can extend the core template to override some blocks (which is exactly why they have blocks).
Replacing the core templates solves a different use case (and in this case, you need to replace the whole template, as trying to extend would extend yourselves)

Reply:
We have already solved that problem by adding another namespace with the prefix `!`, we could do the same here.

Reply:
Reworked the theme feature based on this feedback (I will update the description accordingly).